src/ui/public/images/kibana.svg

```diff
@@ -1,110 +1 @@
s-1" d="M433.46,281.75l0.55,0.42Z" transform="translate(-408.92 -254.55)"/><path class="cls-1" d="M432,280.64l0.31,0.22Z" transform="translate(-408.92 -254.55)"/><path class="cls-1" d="M434.8,282.83l-18.89,22.79h32.26A39.14,39.14,0,0,0,434.8,282.83Z" transform="translate(-408.92 -254.55)"/><path class="cls-2" d="M431.77,280.51c-1.36-1-3.38-2.07-3.38-2.07a39.24,39.24,0,0,1,3.57,2.21Z" transform="translate(-408.92 -254.55)"/><path class="cls-2" d="M434.8,282.82s-0.3-.25-0.79-0.64l0.79,0.65h0Z" transform="translate(-408.92 -254.55)"/><path class="cls-2" d="M432.27,280.87c0.41,0.29.8,0.58,1.2,0.89Z" transform="translate(-408.92 -254.55)"/><path class="cls-2" d="M434,282.18l-0.55-.42c-0.39-.3-0.79-0.6-1.2-0.89l-0.31-.22a39.24,39.24,0,0,0-3.57-2.21l-19.47,23.45v3.73h7l18.89-22.79Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M409.65,273.67l-0.74,0v28.2l19.47-23.45A39,39,0,0,0,409.65,273.67Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M456.2,291.34h-2.62V266.25h2.62v14.81l2.73-.14,5-7.06h3l-5.7,8,5.94,9.43h-3L459,283.23l-2.76.1v8Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M470.31,269.92v-3h2.62v3h-2.62Zm0,21.42V273.87h2.62v17.47h-2.62Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M485.69,273.52q3.53,0,4.94,2t1.42,7q0,5-1.76,7.08t-6.48,2q-1.47,0-4.86-.28l-1-.1V266.25h2.59v8.49A12.25,12.25,0,0,1,485.69,273.52Zm-1.92,15.83q3.42,0,4.52-1.55t1.1-5.26q0-3.7-.87-5.19a3.3,3.3,0,0,0-3.11-1.48,12.22,12.22,0,0,0-4.16.77l-0.7.24v12.3Q482.82,289.35,483.76,289.35Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M508.68,279.14v8.94q0.1,1.29,2,1.54l-0.11,2.06a5.64,5.64,0,0,1-4.16-1.4,15.37,15.37,0,0,1-6.29,1.4,4.75,4.75,0,0,1-3.67-1.36,5.56,5.56,0,0,1-1.26-3.91,4.91,4.91,0,0,1,1.29-3.76,6.86,6.86,0,0,1,4.05-1.48l5.49-.52v-1.5a3.52,3.52,0,0,0-.77-2.55,2.83,2.83,0,0,0-2.1-.77,50.35,50.35,0,0,0-5.8.35l-1.08.11-0.1-2a35.25,35.25,0,0,1,6.81-.77A4.87,4.87,0,0,1,508.68,279.14Zm-10.76,7.13q0,3.22,2.66,3.21a14.13,14.13,0,0,0,4.68-.8l0.8-.28v-5.77l-5.17.49a3.37,3.37,0,0,0-2.27.91A3.21,3.21,0,0,0,497.91,286.27Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M516.47,291.34h-2.62V273.87h2.59v1.22a11.17,11.17,0,0,1,5.42-1.57q3.49,0,4.7,1.89t1.21,6.71v9.22h-2.59v-9.15a11.81,11.81,0,0,0-.72-5q-0.72-1.34-3.06-1.35a9.13,9.13,0,0,0-2.36.33,11.82,11.82,0,0,0-1.9.65l-0.66.31v14.19Z" transform="translate(-408.92 -254.55)"/><path class="cls-3" d="M544.94,279.14v8.94q0.1,1.29,2,1.54l-0.11,2.06a5.64,5.64,0,0,1-4.16-1.4,15.37,15.37,0,0,1-6.29,1.4,4.75,4.75,0,0,1-3.67-1.36,5.56,5.56,0,0,1-1.26-3.91,4.91,4.91,0,0,1,1.29-3.76,6.86,6.86,0,0,1,4.05-1.48l5.49-.52v-1.5a3.52,3.52,0,0,0-.77-2.55,2.83,2.83,0,0,0-2.1-.77,50.35,50.35,0,0,0-5.8.35l-1.08.11-0.1-2a35.25,35.25,0,0,1,6.81-.77A4.87,4.87,0,0,1,544.94,279.14Zm-10.76,7.13q0,3.22,2.66,3.21a14.13,14.13,0,0,0,4.68-.8l0.8-.28v-5.77l-5.17.49a3.37,3.37,0,0,0-2.27.91A3.21,3.21,0,0,0,534.18,286.27Z" transform="translate(-408.92 -254.55)"/></svg>
```

Comment:
The original for this was made in inkscape to ensure it remained editable with open source tooling. This is what the new one looks like in Inkscape. What changed here? I can help redo this in Inkscape if needed.

![screen shot 2016-03-24 at 12 14 06 pm](https://cloud.githubusercontent.com/assets/1250387/14028478/fa97b412-f1b9-11e5-80bf-7c484ee57ab1.png)